control-plane/catalog/to-consul/consul_node_services_client_ent_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build enterprise
```

Comment:
I'm sure there is a reason but why do we need the duplicate enterprise build declaration? Is the old one left for backward compatibility?

Reply:
This stack overflow was good context on how they interplay-- https://stackoverflow.com/questions/68360688/whats-the-difference-between-gobuild-and-build-directives?noredirect=1&lq=1

We can leave both which in theory would be backwards compatible but we have bumped the min go version to 1.17 anyways so we could probably just remove the old `// +build` lines. I think that's probably the cleanest thing to do, does that sound good?

Reply:
Thanks for the info and link!!

I was thinking that if we're bumping the min go version there would be no need for back compatibility lines? If its too much work there's very little to be gained from it, but might be cleaner. Will defer to to your judgement!

Reply:
yup I agree!